apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -320,6 +328,11 @@ export default class Simple2Sequencer extends Sequencer {
   }
 
   private addNewEvent<T extends PlaybackEvent>(event: T) {
+    if (this.currentEventCount >= MAX_NUMBER_EVENTS) {
+      return;
+    } else {
+      this.currentEventCount++;
```

Comment:
minor: should we log a warning the first time we cross the max threshold (i.e. when `this.currentEventCount === MAX_NUMBER_EVENTS`) for some visibility into when this is happening? Additionally/alternatively, should we store a flag on the sequencer output for if the limit was crossed so we can (eventually) give some feedback to the user?

Reply:
Good idea.  It now logs to the console, and also keeps counting which could be useful to examine separately.

I'll hold off storing a flag until we think through a potential user experience.